src/main/java/com/Project/accountManager/controllers/AccountController.java

```diff
@@ -0,0 +1,62 @@
+package com.Project.accountManager.controllers;
+
+import com.Project.accountManager.entities.Account;
+import com.Project.accountManager.request.AccountCreateRequest;
+import com.Project.accountManager.request.AccountDepositRequest;
+import com.Project.accountManager.request.AccountUpdateRequest;
+import com.Project.accountManager.request.AccountWithdrawalRequest;
+import com.Project.accountManager.services.AccountService;
+import com.Project.accountManager.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/account")
+public class AccountController {
+
+    @Autowired
+    AccountService accountService;
+    @Autowired
+    UserService userService;
+
+    public AccountController(UserService userService, AccountService accountService) {
+        this.userService = userService;
+        this.accountService = accountService;
+    }
+
+    @GetMapping
+    public List<Account> getAllAccount(@RequestParam Optional<Long> userId) {
```

Comment:
I see APIs are returning the Account entity.
Let's imagine, the account entity contains sensitive data, such as passwords, confidential investment information, etc...
So is that safe to return all sensitive fields from API?
I highly recommend you research "What is DTO"
https://konstantinmb.medium.com/dtos-101-the-what-why-and-how-of-data-transfer-objects-304a03a71a2c

Reply:
OK, necessary arrangements will be made.